pushapi/identity.go

```diff
@@ -0,0 +1,5 @@
+package pushapi
+
+// Identity represents the data structure of a Coveo identity
+type Identity struct {
+}
```

Comment:
Like a TODO comment? I just have no implemented the Identities functions yet so this is just so it compiles :)